packages/components/src/VirtualizedList/ListTable/ListTable.component.js

```diff
@@ -24,10 +24,12 @@ function ListTable(props) {
 		isSelected,
 		onRowClick,
 		onRowDoubleClick,
+		totalRowCount,
+		rowRenderer,
 		...restProps
 	} = props;
 
-	let RowTableRenderer = DefaultTableRowRenderer;
+	let RowTableRenderer = rowRenderer || DefaultTableRowRenderer;
```

Comment:
We want to be able to render rows even if data is missing, hence the need to use a custom renderer.